packages/dnlib.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,29 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+try {
+  $toolName = 'dnlib'
+  $category = 'dotNet'
+
+  $zipUrl = 'https://www.nuget.org/api/v2/package/dnlib/4.0.0'
+  $zipSha256 = 'adee956696461c9146da3ba220a1a3e6f553a8ba26f2664b1d8507a35976961e'
+  $toolDir = Join-Path ${Env:RAW_TOOLS_DIR} $toolName
+
+  # Download and unzip
+  $packageArgs = @{
+      packageName    = ${Env:ChocolateyPackageName}
+      unzipLocation  = $toolDir
+      url            = $zipUrl
+      checksum       = $zipSha256
+      checksumType   = 'sha256'
+      SpecificFolder = "lib"
+  }
+  Install-ChocolateyZipPackage @packageArgs
+  VM-Assert-Path $toolDir
+
+  $executablePath = Join-Path $toolDir "lib"
```

Comment:
Is this how we want to install dnlib? This creates a link int the FLARE-VM Tools folder to a folder with the following libraries:
![Screenshot 2023-09-05 at 08 06 11](https://github.com/mandiant/VM-Packages/assets/16052290/f78f2fc8-3d6b-4f3b-906f-a605224fac66)

I am not sure if we need the link in the Tools folder as this is not an executable we can run. I am also not sure if copying the libraries somewhere could be more useful. @mike-hunhoff can you help us here? how are you using dnlib and where would you like to have it installed?

Reply:
This looks fine to me! My uses cases for `dnlib` installed in this manner are to compile smaller projects directly via the C# compiler (`csc.exe`) and interactive use via the C# REPL. The important part here is that the `dnlib` DLL is easy to find.

Reply:
Good, then let's merge this! 👍  Thanks @mike-hunhoff